test/unit/api/python/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+#####################
+## CMakeLists.txt
+## Top contributors (to current version):
+##   Makai Mann and Yoni Zohar
+## This file is part of the CVC4 project.
+## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+## in the top-level source directory and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.
+##
+#-----------------------------------------------------------------------------#
+# Add Python bindings API tests
+
+# Check if the pytest Python module is installed.
+execute_process(
+  COMMAND
+  ${PYTHON_EXECUTABLE} -c "import pytest"
+  RESULT_VARIABLE
+    RET_PYTEST
+  ERROR_QUIET
+)
+
+if(RET_PYTEST)
+    message(FATAL_ERROR
+      "Could not find Python module pytest. Install via `pip install pytest'.")
+endif()
```

Comment:
I'd also suggest to change this message to something similar to https://github.com/CVC4/CVC4/blob/master/src/options/CMakeLists.txt#L20 to use `${PYTHON_EXECUTABLE} -m pip install pytest`. Maybe we can add a helper to print this kind of Python module errors.